modules/cli/src/main/java/org/apache/ignite/cli/deprecated/spec/NodeCommandSpec.java

```diff
@@ -100,12 +123,36 @@ public Integer call() {
             out.println(tbl);
             return 0;
         }
+
+        private Path getConfigPath() {
+            return configOptions != null ? configOptions.configPath : null;
+        }
+
+        private String getConfigStr() {
+            if (configOptions == null || configOptions.configPath != null) {
+                return null;
+            }
+            // port is required
+            StringBuilder sb = new StringBuilder();
```

Comment:
I think it is better to use `Config` class from `typesafe` lib to construct a configuration representation.